### DISCUSSION TITLE
Doc comment on Community clients (/client-libraries/community)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1830] [Docs/blog comments]
# Doc comment on Community clients (/client-libraries/community)

Weaviate supports client libraries for these languages:

https://weaviate.io/developers/weaviate/client-libraries/community

**Comment by noizu** [comment]
I wrote an elixir api wrapper a few months ago, which I'll be updating a bit later this quarter: https://github.com/noizu-labs/elixir-weaviate
